applications/nrf5340_audio/README.rst

```diff
@@ -861,6 +861,13 @@ Entering the DFU mode
 The |NCS| uses :ref:`SMP server and mcumgr <zephyr:device_mgmt>` as the DFU backend.
 Unlike the CIS and BIS modes for gateway and headsets, the DFU mode is advertising using the SMP server service.
 For this reason, to enter the DFU mode, you must long press **BTN 4** during each device startup to have the nRF5340 Audio DK enter the DFU mode.
+To distinguish the device role, DFU mode advertising name will be based on role and the pattern : ``CONFIG_BT_DEVICE_NAME`` _ **ROLE** _DFU.
+The name will look like:
+
+* BIS Gateway: Broadcast Audio Source_GW_DFU
```

Comment:
We might need to sync up the ADV name between BIS and CIS for making it consistency.

Reply:
Agree, I don't think it's necessary to differentiate between BIS and CIS. All should be NRF5340_AUDIO_...

Reply:
Are we going to make all BT_DEVICE_NAME the same in this PR?
Like remove [here](https://github.com/nrfconnect/sdk-nrf/blob/main/applications/nrf5340_audio/src/bluetooth/Kconfig.defaults#L197) and [here](https://github.com/nrfconnect/sdk-nrf/blob/main/applications/nrf5340_audio/src/bluetooth/Kconfig.defaults#L230).

Reply:
I suggest that all the headset in DFU mode sharing the same name "NRF5340_AUDIO_GW_DFU"
And all the gateway in DFU mode sharing the same name "NRF5340_AUDIO_XX_DFU"

Reply:
Resolving this conversation. We have agreed that we want to make all device types have the same BT_DEVICE_NAME in this PR.